fetch-content.sh

```diff
@@ -9,7 +9,7 @@ else
 	DIR="$HOME/.openra/Content/ra2/"
 fi
 
-if [ ! -e "$DIR" ]; then
+if [ -z "$(ls -A "$DIR")" ]; then
```

Comment:
Also, regarding validity; it would probably be better to test for a specific expected file rather than checking for any arbitrary file with `ls -A`.